addons/sale/wizard/res_config_settings_views.xml

```diff
@@ -69,8 +69,69 @@
+                                    />
+                                    <button
+                                        string="Activate Razorpay"
+                                        name="action_enable_payment_provider"
+                                        type="object"
+                                        class="btn-primary"
+                                        invisible="onboarding_payment_provider != 'razorpay'"
+                                    />
+                                    <div
+                                        title="Stripe Connect is not available in your country, please use another payment provider."
+                                        invisible="onboarding_payment_provider in ('stripe',
+                                            'razorpay')"
+                                    >
+                                        <button
+                                            string="Activate Stripe"
+                                            class="btn-primary"
+                                            disabled=""
+                                        />
+                                    </div>
+                                </div>
+                                <button
+                                    name='%(payment.action_payment_provider)d'
+                                    icon="oi-arrow-right"
+                                    type="action"
+                                    string="View Alternatives"
+                                    class="btn-link oe_inline"
+                                />
+                            </div>
+                            <div
+                                class="mt8"
+                                invisible="not portal_confirmation_pay or
+                                    providers_state != 'primary_provider'"
+                            >
+                                <button
+                                    name="action_configure_primary_provider"
+                                    type="object"
+                                    class="btn-primary col-auto"
+                                >
+                                    Configure
+                                    <field
+                                        name="onboarding_payment_provider"
+                                        nolabel="1"
+                                        class="oe_inline"
+                                    />
+                                </button>
+                                <button
+                                    type="action"
+                                    name="%(payment.action_payment_provider)d"
+                                    string="View other providers"
+                                    class="btn-link col-auto"
+                                    icon="oi-arrow-right"
+                                />
```

Comment:
Attribute order should follow our guidelines. string,id,name... (apply same for other buttons)